posts/lets_talk_about_9_to_5.md

```diff
@@ -0,0 +1,19 @@
+# Let's talk about 9 to 5
+
+We have a flexible policy towards software engineering work hours. Preferring our engineers to be responsible and ensure that their commitments are met rather than measuring work hours has always been our policy. In this blog post I'd like to argue that measuring work hours can indeed be sensible for measuring a teams effectiveness.
+
+For me it all comes back to the idea that symptoms and illnesses are inextricably linked but while you can detect illnesses via their symptoms, you cure them by addressing the underlying sickness.
+
+In this case work hours are the symptom and the illness is a teams effectiveness or delivery capability. In an environment where there are flexible working hour policies, i.e. choose your own hours, teams frequently working less than 8 hours are likely to be under-challenged. The hours here are the symptom and you're unlikely to solve the problem by enforcing work hours.
+
+Why is a team likely under-challenged if they regularly work less than 8 hours a week? My current answer to this is because we have a social norm of what office hours look like. Regardless of your flexible work hour policy, the rest of your organisation is likely working an 8 hour day which means there will be some level of social pressure around that 8 hour window. If a team is being challenged, they'll usually fill that socially normal window of 8 hours at which point they feel like they've put enough into their day and unless there's an emergency they'll likely feel justified in going home.
```

Comment:
"The rest of your organisation is likely working an 8 hour day" --> meaning only delivery teams benefit from flexible hours? I don't think an organisation who adopts flexible hours would contain it to one team/department/function whatever it is called. 
I believe the pressure comes mostly from the other organisations you work with. Clients, suppliers, or any third party really, who might need you unexpectedly and will expect you to be available during this classic 9 to 5 (id even argue its more 9 to 6)